reliquery/relic.py

```diff
@@ -247,6 +250,15 @@ def get_image(self, name: str) -> BytesIO:
         self.assert_valid_id(name)
         return self.storage.get_binary_obj([self.relic_type, self.name, "images", name])
 
+    def display_image(self, name: str) -> Image:
```

Comment:
It doesn't look like we are actually returning an Image object. I believe if we return `image` we would be returning an Image obj and I believe jupyter lab knows how to render the Image object.

Reply:
I tried messing around with Pillow Image module and found we can just return the image object. We can replace the image.show() with a return statement.

![image](https://user-images.githubusercontent.com/46462993/155801044-00a53b30-a596-4289-af52-3d4e077e5974.png)